src/core/packages/chrome/layout/core-chrome-layout-components/application/layout_application.tsx

```diff
@@ -30,7 +31,12 @@ export const LayoutApplication = ({
   topBar?: ReactNode;
   bottomBar?: ReactNode;
 }) => {
-  const overflow = useEuiOverflowScroll('y');
+  // only restrict overflow scroll on screen (not print) to allow for full page printing
+  const overflow = css`
+    @media screen {
+      ${useEuiOverflowScroll('y')};
```

Comment:
With this, we prevent the application container from scrolling on the grid layout. The scroll moves to the window, and browser printing works as expected.